src/util/util.ts

```diff
@@ -37,3 +37,30 @@ export const isSubstringOf = (substring: string, source: string): boolean =>
   pipe(source, S.toLowerCase, S.includes(S.toLowerCase(substring)))
 
 export const isError = (value: unknown): value is Error => value instanceof Error
+
+export const escapeSparqlStr = (str?: string): string => {
+  const escapeCharactersMap: Record<string, string> = {
+    '\t': '\\t',
+    '\n': '\\n',
+    '\r': '\\r',
+    '\b': '\\b',
+    '\f': '\\f',
+    '"': '\\"',
+    "'": "\\'",
+    '\\': '\\\\'
+  }
+
+  return pipe(
+    str,
+    O.fromNullable,
+    O.map(s =>
+      pipe(
```

Comment:
Leverage `flow` operator to take advantage  [η-reduction](https://sookocheff.com/post/fp/eta-conversion/).

Reply:
Thanks for the link!